__testUtils/mockSession.js

```diff
@@ -32,19 +33,28 @@ export const storageAclUrl = "http://example.com/.acl";
 export const anotherUsersStorageUrl = "http://anotheruser.com/";
 export const anotherUsersStorageAclUrl = "http://anotheruser.com/.acl";
 
+const profileTurtle = oldProfileTurtle;
 export { profileTurtle };
+export { newProfileTurtle };
+
+function getProfileFile(version) {
```

Comment:
I don't think we should tie this to versions. We could base this around whether there is an available editable profile, whether WebID Profile directly or in a separate profile document (as it happens to be the case for ESS 1.2). The files represent the WebID document, that can contain profile information in itself or link to a separate resource containing profile information.